### PR TITLE
Update of termbench with scripts to create plots

### PR DESCRIPTION
**Pull request by Yaraslaut**
Added some tests to check throughput of terminal more reliably and script to create plots

### CODE REVIEW

File: tb/main.cpp [hunk 1]
```diff
@@ -13,11 +13,13 @@
...
 #include <libtermbench/termbench.h>
-#include <iostream>
+
 #include <cstdio>
 #include <cstdlib>
+#include <format>
+#include <iostream>
 #include <string_view>
-#include <fmt/format.h>
+#include <fstream>
```
Comment (christianparpart) [inline tb/main.cpp:22]: shouldn't clang-format have alphabetically sorted this group of includes? 🤔

File: .clang-format [hunk 2]
```diff
@@ -0,0 +1,81 @@
+---
```
Comment (christianparpart) [inline .clang-format:1]: We could copy'n'paste the clang-format CI check into this repo, to have formatting validated right along with adding this file :)

File: libtermbench/termbench.cpp [hunk 3]
```diff
@@ -103,263 +101,274 @@ void Benchmark::runAll()
...
+    os << std::format("{:>40}: {:>3}.{:04} seconds, {}/s (normalized: {}/s)\n",
+                      "all tests",
+                      totalTime.count() / 1000,
+                      totalTime.count() % 1000,
+                      sizeStr(bps),
+                      sizeStr(bps / static_cast<double>(gridCellCount)));
     os << "\n";
-    os << fmt::format(" screen size: {}x{}\n", width_, height_);
-    os << fmt::format("   data size: {}\n", sizeStr(static_cast<double>(testSizeMB_ * 1024 * 1024)));
+    os << std::format(" screen size: {}x{}\n", width_, height_);
```
Comment (christianparpart) [inline libtermbench/termbench.cpp:135]: while I am not against it, would that still work with older Ubuntu versions? 🤔 

I think mixing fmt::format along with std::format use will at least not hurt the final binary. hmmm....

Comment (Yaraslaut) [reply libtermbench/termbench.cpp:135]: We do not use fmt::format here anymore

File: tb/main.cpp (hunk 1 above)
Comment (Yaraslaut) [reply tb/main.cpp]: fixed :)

**Review by christianparpart — APPROVED** [review]
Thank you, @Yaraslaut :)

**Pull request merged**